spec/bifrost/topic_spec.rb

```diff
@@ -56,16 +61,10 @@
   context 'for a defined topic' do
     let(:new_topic) { Bifrost::Topic.new('test_donotdelete') }
 
-    before(:all) do
-      bus = Bifrost::Bus.new
-      tp = Bifrost::Topic.new('test_donotdelete')
-      tp.save unless bus.topic_exists?(tp)
-    end
-
-    after(:all) do
+    before(:each) do
       bus = Bifrost::Bus.new
-      tp = Bifrost::Topic.new('test_donotdelete')
-      tp.delete if bus.topic_exists?(tp)
+      bus.topics.each(&:delete)
```

Comment:
@shirren This logic sounds like it should be global spec functionality. I will put this in the spec helper